catalog/spatial/wps/spatial-wps-endpoint/src/main/java/org/codice/ddf/spatial/ogc/wps/process/endpoint/WpsExceptionMapper.java

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.spatial.ogc.wps.process.endpoint;
+
+import static java.net.HttpURLConnection.HTTP_BAD_REQUEST;
+import static java.net.HttpURLConnection.HTTP_INTERNAL_ERROR;
+
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.ext.ExceptionMapper;
+
+import org.codice.ddf.spatial.ogc.wps.process.api.WpsException;
+import org.codice.ddf.spatial.process.api.ProcessInputDataException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.opengis.ows.v_2_0.ExceptionReport;
+import net.opengis.ows.v_2_0.ExceptionType;
+
+public class WpsExceptionMapper implements ExceptionMapper<Throwable> {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WpsExceptionMapper.class);
+
+    private static final String SERVICE_EXCEPTION_REPORT_VERSION = "2.0.0";
+
+    @Override
+    public Response toResponse(Throwable e) {
+
+        WpsException wpsException;
+        if (e instanceof WpsException) {
+            wpsException = (WpsException) e;
+            LOGGER.debug("%s: %s. %s",
+                    wpsException.getExceptionCode(),
+                    wpsException.getMessage(),
+                    wpsException.getLocator(),
+                    e);
+        } else if (e instanceof ProcessInputDataException) {
+            String inputIds = String.join(",", ((ProcessInputDataException) e).getInputIds());
+            LOGGER.debug("%s: %s", e.getMessage(), inputIds, e);
+            wpsException = new WpsException(
+                    "One or more of inputs for which the service was able to retrieve the data but could not read it.",
+                    "WrongInputData",
+                    inputIds,
+                    HTTP_INTERNAL_ERROR);
```

Comment:
This should be `HTTP_BAD_REQUEST` and the `InternalServerError` should be `HTTP_INTERNAL_ERROR`.